cms/static/cms/js/modules/cms.tooltip.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright https://github.com/divio/django-cms
+ */
+
+// #############################################################################
+// NAMESPACES
+/**
+ * @module CMS
+ */
+var CMS = window.CMS || {};
+
+// #############################################################################
+// Tooltip
+(function ($) {
+    'use strict';
+
+    // shorthand for jQuery(document).ready();
+    $(function () {
+        /**
+         * The tooltip is the element which shows over plugins
+         * and suggests clicking/tapping them to edit.
+         *
+         * @class Tooltip
+         * @namespace CMS
+         */
+        CMS.Tooltip = new CMS.Class({
+
+            initialize: function () {
+                this.isTouch = false;
+                this.domElem = this.pick();
+
+                this.checkTouch();
+            },
+
+            checkTouch: function () {
```

Comment:
can you add jsdoc for all methods, like in https://github.com/divio/django-cms/blob/preview/cms/static/cms/js/modules/cms.modal.js ?